README.md

```diff
@@ -44,59 +50,53 @@ cd generative_data_prep
+The following simple example will help you get started with your first processed dataset. Here are a few important parameters to know about when running this example:
 
-## Input format
+|          Flag Name          | Type | Description | Instructions |
+|             ---             | ---  |     ---     |      ---     |
+|   `input_file_path`         |  str | An existing file path to the dataset to be processed. File must be in `.jsonl` or `.txt` format.   |  Check out the [input format](#input-format) section for more details.    |
+|   `output_path`             |  str | A path to the desired output location for the directory of processed dataset files. If the path doesn't exist, a new directory will be created using the provided path.   |   Processed datasets consist of multiple files under an output directory. If I want my output directory to be named `out`, I could put the path `/Users/johndoe/Documents/datasets/dataset_name/out` for example. Check out the [output](#output) section for more details. |
+|   `pretrained_tokenizer`    |  str | The tokenizer to use when tokenizing the input dataset. The tokenizers are model specific and can be found on HuggingFace.      |  You can get this value by copying the model path available on the HuggingFace model card. I.e. For Llama-2-7b-hf I would put `"meta-llama/Llama-2-7b-hf"`  |
+|   `max_seq_length`          |  int | The max size of input sequence a model can support. This is model specific - i.e. for `GPT-2` it's __1024__, for `Llama-2` it's __4096__.  |   You can find this information in a few places, but a place you can consistently find this value is in the `config.json` file under the HuffingFace model's File's and Versions tab. Then grab the value for `max_position_embeddings`.  |
+|   `input_packing_config`    |  str | Defines the strategy used to pack the provided data into sequences across the output HDF5 files. |   There are 7 options for this flag: `'full'`, `'single::truncate_left'`, `'single::truncate_right'`, `'single::drop'`, `'greedy::truncate_left'`, `'greedy::truncate_right'`, `'greedy::drop'`. Check out the [`input_packing_config` flag](#input_packing_config) below for an in depth description of these options. |
+|          `shuffle`          |  int | Determines whether to shuffle the input dataset, and whether to shuffle on RAM. |   Check out the [shuffle flag](#shuffle) below for more details.  |
 
-Each line in the input file must be either plain text or [jsonline](https://jsonlines.org/). if the jsonline has different keywords, refer to the `prompt_keyword`, and `completion_keyword` flag documentation below.
+### Example
+```shell
+python3 -m generative_data_prep pipeline --input_file_path=<PATH TO DATASET FILE> --output_path=<PATH TO OUTPUT DIRECTORY> --pretrained_tokenizer=gpt2 --max_seq_length=1024 --input_packing_config='single::drop' --shuffle=on_RAM
```

Comment:
I would say that 9/10 users going through this page will be doing so with the intention of fine-tuning as opposed to pre-training due to the amount of data required for pre-training. Because of this, I added in the `--input_packing_config='single:drop' in this example since I believe it's a better option for fine-tuning - from what I've seen, most users individual completions stay within the max_seq_len 3/4 times so that's why I opted for drop as opposed to truncating. Could be worth discussing and potentially changing the default value depending on your thoughts here!

Reply:
Ah ya good point, I'll make the change!